### PR TITLE
Fix Dockerfile

### PR DESCRIPTION
**Pull request by WiscEvan**
- :art::bug::green_heart::penguin: Fix Dockerfile. Now capable of running autometa entrypoints within docker container
- Fixes issue #3

### CODE REVIEW

**Review by chasemc — COMMENTED** [review]
Consider adding at least minimum versions for the Conda install libraries

File: Dockerfile [hunk 1]
```diff
@@ -19,15 +19,45 @@ MAINTAINER Jason C. Kwan "jason.kwan@wisc.edu"
...
+RUN autometa-length-filter -h \
+    & autometa-orfs -h \
+    & autometa-coverage -h \
+    & autometa-kmers -h \
+    & autometa-markers -h \
+    & autometa-taxonomy -h \
+    & autometa-taxonomy-lca -h \
+    & autometa-taxonomy-majority-vote -h \
+    & autometa-binning -h \
+    & autometa-unclustered-recruitment -h
```
Comment (jason-c-kwan) [inline Dockerfile:63]: The output of this only shows up when you build the docker image, not when you run it. So no one will see this. Perhaps put this in the bash rc file?

**Review by jason-c-kwan — CHANGES_REQUESTED** [review]
Dockerfile builds and runs on my local machine. One issue noted to do with the help message only showing up during build and not run. Please note: I did not do extensive testing of the pipeline in the docker image. The Docker hub build will only really be tested when this is merged (due to the Docker hub settings), but I predict it will at least build.

File: Dockerfile (hunk 1 above)
Comment (WiscEvan) [reply Dockerfile:63]: Do you want the help text for all of the entrypoints to be printed to the user each time a container is spawned from the image? This is a minimal image as it does not have any of the required databases configured. The user will need to mount their databases to the docker container and then have to point autometa to these mounted paths. That being said, I think nextflow can be configured such that the user will not have to worry about issuing these docker commands. We can just hardcode the docker image into each process and nextflow will do the rest.

Comment (jason-c-kwan) [reply Dockerfile]: Oh, I see. It looked like your intention was to show the messages to the user. In that case, you could probably take them out after showing it works. On another note though, if the Docker is intended to be used interactively, then a message upon startup might be a good idea.

**Review by jason-c-kwan — APPROVED** [review]
Looks good, the image builds on my local machine.

**Pull request merged**